src/tex/appendix.tex

```diff
@@ -166,20 +166,36 @@ \subsection{Model performance} \label{sec:model_performance_app}
 \begin{figure}[htb]
     \centering
     \includegraphics{figures/overall_performance.pdf}
-    \caption{\textbf{Overall performance of the models on the \chembench corpus.} The bar plot shows the fraction of questions that were answered completely correctly by the models. Scores computed on the entire \chembench corpus.}
+    \caption{\textbf{Overall performance of the models on the \chembench corpus.} The bar plot shows the fraction of questions that were answered correctly by the models. Scores computed on the entire \chembench corpus.}
     \label{fig:barplot_all_correct_all_questions}
     \script{plot_overview_performance_plot.py}
 \end{figure}
 
 \Cref{fig:all_questions_models_completely_correct_radar_overall} shows the performance of the models on the different topics of the \chembench corpus.
 The general pattern of performance varies significantly between the different topics and is also observed when the models are evaluated on the entire corpus.
-However, since some subjects are composed of questions from different sources, the ranking of the models is, in some instances, different from the one on \chembenchmini.
+However, since some subjects are composed of questions from different sources, the ranking of the models is, in some instances, different from the one on the \enquote{tiny} subset.
+
+\begin{table}
+    \caption{\textbf{Performance of the models on the \chembench corpus.} The table shows the fraction of questions answered correctly by the models for different skills and difficulty levels.}
+    \resizebox{\textwidth}{!}{
+    \variable{output/performance_table.tex}
+    }
+    \label{tab:performance_table}
+\end{table}
+
+\begin{table}
+    \caption{\textbf{Performance of the models on the \enquote{tiny} subset of the \chembench corpus.} The table shows the fraction of questions answered correctly by the models for different skills and difficulty levels.}
+    \resizebox{\textwidth}{!}{
+    \variable{output/performance_table_human_subset.tex}
+    }
+    \label{tab:performance_table_human_subset}
+\end{table}
```

Comment:
here, also the `tiny` stuff should be `\chembenchmini` as it currently is on main